gef-ui/server/controllers/amendments/helpers/amendment-summary-view-model.helper.ts

```diff
@@ -128,19 +129,21 @@ const generateCoverEndDateSummaryRows = (amendment: PortalFacilityAmendmentWithU
  * @param renderChangeLink - if change link should be rendered - defaults to true
  * @returns the SummaryListRows containing `New facility value`
  */
-const generateFacilityValueSummaryRows = (amendment: PortalFacilityAmendmentWithUkefId, renderChangeLink = true): SummaryListRow[] => {
+const generateFacilityValueSummaryRows = (amendment: PortalFacilityAmendmentWithUkefId, facility: Facility, renderChangeLink = true): SummaryListRow[] => {
   if (!amendment.changeFacilityValue) {
     return [];
   }
 
+  const currencyValue = amendment.value ? `${facility.currency?.id.toString()} ${getFormattedMonetaryValue(Number(amendment.value))}` : '-';
```

Comment:
const currency = facility.currency?.id.toString()
const value = amendment.value ?? getFormattedMonetaryValue(Number(amendment.value));

```suggestion
  const currencyValue = amendment.value ? `${currency} ${value}` : '-';
```

What do you think? easier to follow?

Reply:
Good spot, have refactored.